### DISCUSSION TITLE
How can I install previous version of the ComfyUI IPAdapter Plus in ComfyUI?

### DISCUSSION THREAD

**Discussion by nurettn** [cubiq/ComfyUI_IPAdapter_plus#543] [Q&A]
Some nodes are missing from the tutorial that I want to implement. It seems some of the nodes were removed from the codebase [like in this issue](https://github.com/cubiq/ComfyUI_IPAdapter_plus/issues/483) and I'm not able to implement the tutorial. 

How can I roll back to or install the previous version (the version before that was released in May) of ComfyUI IPAdapter Plus?

**Comment by schoenid** [comment]
I've got also the problem, that some node functions stopped working since ~v3.23, so I would like to get my older installation back ...
